specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json

```
@@ -545,7 +569,10 @@
     },
     "/timeseries/instances/search": {
       "post": {
-        "operationId": "SearchInstancesPaged",
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_SearchPaged",
```

I am divided on adding Paged or not - adding it makes it more precise and consistent, on the other hand, Search will never be non-paged. I would probably remove Paged - makes it simpler.
```suggestion
        "operationId": "Instances_Search",
```

---

Or should we drop Paged everywhere?